Challenge1/src/agent.spec.ts

```diff
@@ -1,74 +1,127 @@
-// import {
-//   FindingType,
-//   FindingSeverity,
-//   Finding,
-//   HandleTransaction,
-//   createTransactionEvent,
-//   ethers,
-// } from "forta-agent";
-// import agent, {
-//   ERC20_TRANSFER_EVENT,
-//   TETHER_ADDRESS,
-//   TETHER_DECIMALS,
-// } from "./agent";
+import { Finding, HandleTransaction, FindingSeverity, FindingType, TransactionEvent, ethers } from "forta-agent";
+import { createAddress } from "forta-agent-tools";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { provideHandleTransaction } from "./agent";
+import { CREATE_AGENT_ABI, UPDATE_AGENT_ABI, OTHER_FUNCTION_ABI, CHAIN_IDS } from "./constants";
 
-// describe("high tether transfer agent", () => {
-//   let handleTransaction: HandleTransaction;
-//   const mockTxEvent = createTransactionEvent({} as any);
+const mockNethermindAddress = createAddress("0x01");
+const mockFortaRegistryAddress = createAddress("0x02");
+const mockOtherAddress = createAddress("0x03");
 
-//   beforeAll(() => {
-//     handleTransaction = agent.handleTransaction;
-//   });
+const MOCK_AGENT_ID = 1;
 
-//   describe("handleTransaction", () => {
-//     it("returns empty findings if there are no Tether transfers", async () => {
-//       mockTxEvent.filterLog = jest.fn().mockReturnValue([]);
+const ifaceForSetData = new ethers.utils.Interface([CREATE_AGENT_ABI, UPDATE_AGENT_ABI, OTHER_FUNCTION_ABI]);
+const encodedCreateAgentData = ifaceForSetData.encodeFunctionData("createAgent", [
+  MOCK_AGENT_ID,
+  mockNethermindAddress,
+  "metadata",
+  CHAIN_IDS,
```

Comment:
Would request you declare a `mockChainIds` in this test file as well, like the other arguments.